src/index.html

```diff
@@ -1,19 +1,663 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
     <title>Dia</title>
+
+    <!-- Icons-->
+    <link
+      href="https://cdn.jsdelivr.net/npm/remixicon@4.7.0/fonts/remixicon.css"
+      rel="stylesheet"
+    />
+    <!--Swiper -->
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/swiper@12/swiper-bundle.min.css"
+    />
+    <!--Main Styles -->
     <link
       rel="stylesheet"
       href="./styles/main.scss"
     />
   </head>
-  <body>
-    <h1>Dia</h1>
+
+  <body
+    class="page__body"
+    id="page__body"
+  >
```

Comment:
when you remove class from `html` also please remove class from this
also `id` for body is also unnecessarily because `body` is unique tag